core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -899,8 +920,8 @@ protected int doHashCode() {
         return Objects.hash(queryString, defaultField, fieldsAndWeights, defaultOperator, analyzer, quoteAnalyzer,
                 quoteFieldSuffix, autoGeneratePhraseQueries, allowLeadingWildcard, analyzeWildcard,
                 enablePositionIncrements, fuzziness, fuzzyPrefixLength,
-                fuzzyMaxExpansions, fuzzyRewrite, phraseSlop, useDisMax, tieBreaker, rewrite, minimumShouldMatch, lenient,
```

Comment:
I believe lenient was shifted to the next line because of the character width is set to 140 as specified by the elasticsearch contributing guidelines. Logic remains the same. @Helen-Zhao to confirm